app/views/items/index.html.erb

```diff
@@ -1,13 +1,21 @@
-<h1> Items for Sale </h1>
-<a class="btn btn-success" href="/items/new" role="button">Add Item</a>
-<ul>
-  <% @presenter.items.each do |item| %>
-    <li>
-      <%= item.name %> : <%= item.price %> <%= image_tag(item.image, size: "100") %>
-      <button type="button" class="btn btn-primary btn-sm">Buy Now!</button>
-      <br>
-      <%= item.description %>
-    </li>
-  <% end %>
-</ul>
+<div class="container">
+  <div class="jumbotron">
+    <h1 class="text-center"> Items for Sale </h1>
+    <p class="text-center">  <a class="btn btn-success btn-add-item" href="/items/new" role="button">Add Item</a></p>
+  </div>
+</div>
 
+<div class="container">
+  <% @presenter.available_items.each do |item| %>
+    <div class="col-xs-12 col-md-4 col-lg-3">
+      <div class="row, item-container">
+        <div class="col-md-12, item-details-container">
+          <div  class="items-index-image-container">  <%= image_tag(item.image, size: "100") %> </div>
+          <div class="title-price"> <strong> <%= item.name.capitalize %> : $<%= item.price %> </strong></div>
+          <p> <a class="btn btn-primary btn-sm btn-buy-now" href="items/<%= item.id %>/purchases/new" role="button">Buy Now!</a></p>
```

Comment:
I think [link_to](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) is the Rails way to create this button...?